### PR TITLE
fixed styles

### PR DESCRIPTION
**Pull request by okolomiets**
https://jira.doubledutch.me/browse/STEW-672
https://jira.doubledutch.me/browse/STEW-665
https://jira.doubledutch.me/browse/STEW-664

### CODE REVIEW

**Review by kobvel — APPROVED** [review]

File: src/styles/blocks/input.less [hunk 1]
```diff
@@ -209,11 +209,17 @@ form {
         font-weight: 400;
         box-sizing: border-box;
 
-        &::-webkit-input-placeholder,
-        &::-moz-placeholder,
-        &:-moz-placeholder,
+        &::-webkit-input-placeholder {
```
Comment (kurpav) [inline src/styles/blocks/input.less:212]: why did you changed it?

File: src/styles/components/select.less [hunk 2]
```diff
@@ -31,6 +31,19 @@
 .selectize-control.multi.ng-empty > .selectize-input > input {
     position: relative;
     top: 6px;
+
+    &::-webkit-input-placeholder {
```
Comment (kurpav) [inline src/styles/components/select.less:35]: you can write it shorter

File: src/styles/blocks/input.less (hunk 1 above)
Comment (okolomiets) [reply src/styles/blocks/input.less:212]: because it didn't work via commas

File: src/styles/components/select.less (hunk 2 above)
Comment (okolomiets) [reply src/styles/components/select.less:35]: if you mean as per fixed one above - I can't

**Pull request merged**